packages/zoe/src/contractFacet/rightsConservation.js

```diff
@@ -46,34 +46,43 @@ const assertEqualPerBrand = (leftSumsByBrand, rightSumsByBrand) => {
   // only if the sum for the brand in the other map is empty.
 
```

Comment:
Deleting like 37 above makes the comment clearer.

Reply:
@erights, sorry, I don't understand this comment

Reply:
```js
/**
 * Assert that the left sums by brand equal the right sums by brand
 *
 * @param  {Store<Brand, Amount>} leftSumsByBrand - a map of brands to sums
 * @param  {Store<Brand, Amount>} rightSumsByBrand - a map of brands to sums
 * indexed by issuer
 */
```
Line 37 says "indexed by issuer", only as an extension of "a map of brands to sums" for `rightSumsByBrand`, not `leftSumsByBrand`. What does it mean?

Reply:
oh, I see, you want to delete that line. That makes sense. I think its inclusion is a mistake from past implementations.

Reply:
yes